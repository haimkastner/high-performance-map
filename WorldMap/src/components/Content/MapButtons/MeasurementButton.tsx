import React from "react";
import { MapButtonIcon } from './MapButtonIcon';

export class MeasurementButton extends React.Component {
  render() {
    return (<MapButtonIcon imageName={"adding-copy-shape-3@2x.png"}></MapButtonIcon>);
  }
}
