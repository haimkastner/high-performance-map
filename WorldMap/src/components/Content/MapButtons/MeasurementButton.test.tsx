import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { MeasurementButton } from './MeasurementButton';

describe('MeasurementButton Component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<MeasurementButton />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('matches the snapshot', () => {
    const tree = renderer.create(<MeasurementButton />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  // tslint:disable-next-line: no-empty
  describe('MeasurementButton Component Logic', () => {});
});
