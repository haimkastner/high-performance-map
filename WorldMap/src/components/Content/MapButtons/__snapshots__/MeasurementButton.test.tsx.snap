// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MeasurementButton Component matches the snapshot 1`] = `
<div
  className="MuiBox-root MuiBox-root-4"
>
  <img
    height={25}
    src="adding-copy-shape-3@2x.png"
    width={25}
  />
</div>
`;
